### PR TITLE
Private route

### PR DESCRIPTION
**Pull request by LeoDiduch**

### CODE REVIEW

**Review by BastienSaulnier — DISMISSED** [review]
SnE

**Review by BastienSaulnier — DISMISSED** [review]
SnE

File: frontend/src/components/FormLoginBO.jsx [hunk 1]
```diff
@@ -16,7 +16,10 @@ export default function FormLoginBO() {
     axios
       .post(`${apiCall}/auth/login`, user)
       .then(res => {
-        dispatch({ type: "JWT", value: res.data });
+        dispatch({ type: "JWT", value: res.data.token });
```
Comment (LoicBrassart) [inline frontend/src/components/FormLoginBO.jsx]: (Generally speaking, we would `dispatch` only once, giving an object)

File: frontend/src/pages/HomeBO.jsx [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+import FormLoginBO from "../components/FormLoginBO";
 import "./styles/HomeBO.scss";
+import { useSelector } from "react-redux";
 
 export default function HomeBO() {
-  return <div className="HomeBO"></div>;
+
+  return (
+    <div className="HomeBO">
+      <FormCreateUser />}
```
Comment (LoicBrassart) [inline frontend/src/pages/HomeBO.jsx]: Why this bracket ?

File: frontend/src/redux/reducers/reducer.js [hunk 3]
```diff
@@ -2,54 +2,77 @@ const initialState = {
...
+  isSuperAdmin: null,
+  isAdmin: null,
+  isLogged: false,
+  userId: null
 };
 
 const reducer = (state = initialState, action) => {
-  const newState = JSON.parse(JSON.stringify(state));
   switch (action.type) {
+    case "SUPERADMIN":
```
Comment (LoicBrassart) [inline frontend/src/redux/reducers/reducer.js]: ... And that's why we dispaatch only once, generally speaking

**Review by LoicBrassart — CHANGES_REQUESTED** [review]

File: frontend/src/pages/HomeBO.jsx (hunk 2 above)
Comment (LeoDiduch) [reply frontend/src/pages/HomeBO.jsx]: sorry :+1:

**Review by BastienSaulnier — APPROVED** [review]
SnE

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**